src/openalea/lpy/lsysparameters.py

```diff
@@ -215,8 +215,8 @@ def add(self, name, value, ptype = None, category = None, **params):
             raise TypeError(ptype)
 
     def add_category(self, name, **params):
-        category = Category({ 'name' : category, 'enabled' : True })
-        category.update(params)
+        category = Category({ 'name' : name, 'enabled' : True })
```

Comment:
Thanks for detecting this error.
category = Category({ 'name' : name, 'enabled' : True }, params=params)
would be better.

Reply:
fixed in 10b96a2